mms/tests/integration_tests/onnx_export_integ_test.py

```diff
@@ -34,46 +59,87 @@ def _download_file(download_dir, url):
 
 def cleanup(tmpdir):
     print("Deleting all downloaded resources for SSD MXNet Model Server Integration Test")
-    shutil.rmtree(tmpdir)
+    shutil.rmtree(str(tmpdir))
 
 
-def setup_onnx_integ(tmpdir):
+def setup_onnx_integ(tmpdir, URL,port):
```

Comment:
nit pick: a space " " after "," makes for better readability. In a fleeting glance, this looks like URL.port.

Reply:
I will do that